### PR TITLE
NEW use select2 multiselect for project status

### PR DESCRIPTION
**Pull request by cfoellmann**
Use the select2 multiselect to mix project status

### CODE REVIEW

File: htdocs/projet/list.php [hunk 1]
```diff
@@ -147,8 +146,10 @@
 $search_date_end_end = dol_mktime(23, 59, 59, $search_date_end_endmonth, $search_date_end_endday, $search_date_end_endyear);	// Use tzserver
 $search_category_array = GETPOST("search_category_".Categorie::TYPE_PROJECT."_list", "array");
 
-if ($search_status == '') {
-	$search_status = -1; // -1 or 1
+if (is_array(GETPOST('search_status', 'none'))) {	// 'none' because we want to know type before sanitizing
```
Comment (eldy) [inline htdocs/projet/list.php:149]: You can use GETPOSTISARRAY('search_status') for that.

Comment (cfoellmann) [reply htdocs/projet/list.php]: DONE

**Comment by cfoellmann** [comment]
@eldy this is ready for 17.x now

**Comment by cfoellmann** [comment]
can be merged into 18.x alpha

**Comment by eldy** [comment]
@cfoellmann Can you help me to fix the conflict of this PR ?

**Pull request closed without merging**